src/sentry/seer/utils.py

```diff
@@ -290,3 +291,29 @@ def post_bulk_grouping_records(
         extra.update({"reason": response.reason})
         logger.info("seer.post_bulk_grouping_records.failure", extra=extra)
         return {"success": False}
+
+
+def delete_grouping_records(
+    project_id: int,
+) -> bool:
+    try:
+        response = seer_grouping_connection_pool.urlopen(
+            "GET",
+            SEER_GROUPING_RECORDS_DELETE_URL + f"/{project_id}",
```

Comment:
Trailing slash to avoid redirect, and whole thing as f-string to avoid concatenation
```suggestion
           f"{SEER_GROUPING_RECORDS_DELETE_URL}/ {project_id}/",
```

Reply:
i don't think seer has trailing slashes, at least we don't add them on our other strings